### PR TITLE
Fix warnings

### PR DESCRIPTION
**Pull request by eregon**

### CODE REVIEW

File: src/main/java/org/truffleruby/language/WarningNode.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import com.oracle.truffle.api.CompilerDirectives.TruffleBoundary;
+import com.oracle.truffle.api.object.DynamicObject;
+import com.oracle.truffle.api.source.SourceSection;
+import org.jcodings.specific.UTF8Encoding;
+import org.truffleruby.RubyLanguage;
+import org.truffleruby.core.rope.CodeRange;
+import org.truffleruby.core.string.StringNodes;
+import org.truffleruby.language.dispatch.CallDispatchHeadNode;
+
+/** Warns only if $VERBOSE is true. */
```
Comment (bjfish) [inline src/main/java/org/truffleruby/language/WarningNode.java]: *truthy ?

File: src/main/java/org/truffleruby/language/WarningNode.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2016 Oracle and/or its affiliates. All rights reserved. This
```
Comment (chrisseaton) [inline src/main/java/org/truffleruby/language/WarningNode.java]: 2017?

Comment (eregon) [reply src/main/java/org/truffleruby/language/WarningNode.java]: Right. I'll also do a big copyright update soon.

File: src/main/java/org/truffleruby/language/WarningNode.java (hunk 1 above)
Comment (eregon) [reply src/main/java/org/truffleruby/language/WarningNode.java]: No, `$VERBOSE` can only have 3 values, so it's only the singleton `true`:
```
[1] pry(main)> $VERBOSE=42
=> 42
[2] pry(main)> $VERBOSE
=> true
```

**Pull request merged**